### PR TITLE
Fixes 

### PR DESCRIPTION
**Pull request by Phil9219**

### CODE REVIEW

File: client/src/components/Map/Map.js [hunk 1]
```diff
@@ -22,33 +22,33 @@ import {
...
+//     ::placeholder {
+//       color: var(--inputfield-color);
+//     }
+//   }
+
+//   button:nth-child(2) {
+//     background: none;
+//     border: none;
+//   }
+// `;
```
Comment (mariegross) [inline client/src/components/Map/Map.js]: If you don't need it anymore, I would delete it :)

File: client/src/components/Page/CreatedRunPage.js [hunk 2]
```diff
@@ -8,6 +8,11 @@ import { useParams } from "react-router-dom";
 const CreatedRunPageContainer = styled.div`
   width: 100%;
   height: 100%;
+  margin-top: 30%;
+  margin-bottom: 15%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
```
Comment (mariegross) [inline client/src/components/Page/CreatedRunPage.js]: Looks like this container is basically the same as the ChooseYourRunPageContainer – you could just use this and update with the additional styling :)

File: client/src/components/Page/RunSetupPage.js [hunk 3]
```diff
@@ -43,12 +44,12 @@ export default function RunSetupPage() {
       <RunSetupContainer>
         <Calendar />
 
-        <Map />
+        <Map></Map>
         <InputfieldsContainer>
-          <Input placeholder="Choose Your Distance" />
+          <Input placeholder="Choose Your Kilometers" />
```
Comment (mariegross) [inline client/src/components/Page/RunSetupPage.js]:
```suggestion
          <Input placeholder="Choose Your Distance (in km)" />
```

**Review by mariegross — COMMENTED** [review]
lgtm - it seems like most of your containers / divs are very similar - maybe you don't need a separate component for each, instead you could build one basic container and update it slightly where you need it :)

**Review by arassaria — COMMENTED** [review]
🚀 LGTM 🚀

File: client/db.json [hunk 4]
```diff
@@ -20,6 +20,13 @@
       "date": "26.12.2020",
       "time": "13:30",
       "distance": "7,5"
+    },
+    {
+      "runName": "Aachener Weiher Run",
+      "id": 4,
+      "date": "27.12.2020",
```
Comment (lmachens) [inline client/db.json:27]: Not important for now, but later, you should save dates in ISO-format or JSON (or in Date if you are using MongoDB).
This makes it easier to convert it to a date object and compare it to other dates.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toISOString
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toJSON

File: client/src/components/Page/ChooseYourRunPage.js [hunk 5]
```diff
@@ -35,21 +35,18 @@ export default function ChooseYourRunPage() {
...
-                isFavorite={true}
-                {...run}
-              ></RunCard>
-            </Link>
+            <RunCard
+              key={id}
+              onFavoriteClick={() => alert("click")}
+              isFavorite={true}
+              {...run}
+            ></RunCard>
```
Comment (lmachens) [inline client/src/components/Page/ChooseYourRunPage.js]:
```suggestion
            />
```

File: client/src/components/Page/RunsDonePage.js [hunk 6]
```diff
@@ -32,17 +35,16 @@ export default function RunsDonePage() {
...
-                isFavorite={true}
-                {...run}
-              ></RunCard>
-            </Link>
+            <RunCard
+              key={id}
+              onFavoriteClick={() => alert("click")}
+              isFavorite={true}
+              {...run}
+            ></RunCard>
```
Comment (lmachens) [inline client/src/components/Page/RunsDonePage.js]:
```suggestion
            />
```

**Review by lmachens — COMMENTED** [review]
Good job!! Great improvements here 🎉.

File: client/src/components/Map/Map.js (hunk 1 above)
Comment (Phil9219) [reply client/src/components/Map/Map.js]: Fixed it and put inputfield an button on top of map.
Thanks :)

File: client/src/components/Page/ChooseYourRunPage.js (hunk 5 above)
Comment (Phil9219) [reply client/src/components/Page/ChooseYourRunPage.js]: 👍

File: client/db.json (hunk 4 above)
Comment (Phil9219) [reply client/db.json:27]: Alright. Thanks for the resources. 
I`ll make this a new issue. :)

File: client/src/components/Page/RunSetupPage.js (hunk 3 above)
Comment (Phil9219) [reply client/src/components/Page/RunSetupPage.js]: Good Idea. Thanks

File: client/src/components/Page/CreatedRunPage.js (hunk 2 above)
Comment (Phil9219) [reply client/src/components/Page/CreatedRunPage.js]: Good Idea. Thanks

**Pull request merged**